src/docs/guide/usage/linter/rules/typescript/await-thenable.md

```diff
@@ -61,6 +62,7 @@ await {
 // this is a Promise - produced by calling a function
 declare const getPromise: () => Promise<string>;
 await getPromise();
+await getPromise();
```

Comment:
lol there it is again

Reply:
We need to remove here... 🫠 

https://github.com/oxc-project/oxc/blob/c1e0c89e14fec87b9583cc757da29029d8bb7e7f/crates/oxc_linter/src/rules/typescript/await_thenable.rs#L20